turbiniactl

```diff
@@ -44,23 +44,42 @@ log = logging.getLogger('turbinia')
 logger.setup()
 
 
-def PrintTaskStatus(project, region, days=1, all_fields=False):
+def PrintTaskStatus(instance, project, region, days=0, task_id=None,
+                    request_id=None, all_fields=False):
   """Prints the recent history for Turbinia Tasks.
 
   Args:
+    instance (string): The Turbinia instance name.
     project (string): The name of the project.
     region (string): The name of the zone to execute in.
+    days (int): The number of days we want history for.
+    task_id (string): The Id of the task.
+    request_id (string): The Id of the request we want tasks for.
   """
   function = GoogleCloudFunction(project_id=project, region=region)
-  start = datetime.now() - timedelta(days=days)
-  # Format this like '1990-01-01T00:00:00z' so we can cast it directly to a
-  # javascript Date() object in the cloud function.
-  start_string = start.strftime('%Y-%m-%dT%H:%M:%S')
-  response = function.ExecuteFunction(
-      'getrecenttasks', {'kind': 'TurbiniaTask', 'start':start_string})
+  func_args = {'instance': instance, 'kind': 'TurbiniaTask'}
 
+  if days:
+    start = datetime.now() - timedelta(days=days)
+    # Format this like '1990-01-01T00:00:00z' so we can cast it directly to a
+    # javascript Date() object in the cloud function.
+    start_string = start.strftime('%Y-%m-%dT%H:%M:%S')
+    func_args.update({'start': start_string})
+  elif task_id:
+    func_args.update({'task_id': task_id})
+  elif request_id:
+    func_args.update({'request_id': request_id})
+
+  response = function.ExecuteFunction('gettasks', func_args)
   if not response.has_key('result'):
-    log.error('Return from GCF does not contain a "result" key.')
+    log.error('No results found')
+    print '\nNo results found.\n'
```

Comment:
We should move this to a common function that handles stdout and also logging etc. But fine to do this later.

Reply:
Added #104 for this.